public_html/js/index.js

```diff
@@ -1,104 +1,120 @@
-/**
- * Save a review
- * @param id int ID of the record
- * @param val string Save value 'fixed' or 'false'
- */
-function saveState( id, val ) {
-	var buttonId, unusedButtonId, buttonClass, unusedButtonClass;
+$( document ).ready( function() {
+	/** Listeners */
+	$('body').tooltip({
+		selector: '[data-toggle="tooltip"]'
+	});
+	$( '.records' ).on( 'click', '.js-save-state', function () {
+		saveState( $( this ).data( 'id' ), $( this ).data( 'status' ) );
+	} );
+	$( '.js-load-more' ).on( 'click', loadMoreResults );
+	$( '.js-compare-button' ).on( 'click', function () {
+		// pass the dataset of the element as an object to toggleComparePane
+		toggleComparePane.call(this, this.dataset);
```

Comment:
I merged the branch without this last patch, for toggleComparePane because it doesn't bind to dynamically created records (with load more functionality). I researched around and found this: http://stackoverflow.com/questions/203198/event-binding-on-dynamically-created-elements so we should be binding to the parent and not the dynamically created child. This would be problematic because we can't find individual compare records without the ithenticate ID and loop index.